core/services/relay/evm/evmtesting/chain_components_interface_tester.go

```diff
@@ -323,6 +350,10 @@ func (it *EVMChainComponentsInterfaceTester[T]) GetContractReader(t T) clcommont
 	conf, err := types.ChainReaderConfigFromBytes(confBytes)
 	require.NoError(t, err)
 
+	// Inject the EVMAddressModifier after decoding.
+	// AddressModifier iface is skipped in JSON serialization.
```

Comment:
maybe something like  `// AddressModifier iface is skipped in JSON serialization and should be injected at runtime.`